x-pack/plugins/triggers_actions_ui/public/application/sections/rule_form/rule_form.tsx

```diff
@@ -275,13 +276,9 @@ export const RuleForm = ({
       },
       new Map()
     );
-    const solutionsEntries = [...solutionsResult.entries()];
```

Comment:
By deleting that we are messing up the way we set up the consumer in server-less for o11y

Reply:
This looks like a bad merge conf resolution, I don't remember removing this.